jobqueue/src/main/java/com/path/android/jobqueue/network/NetworkUtilImpl.java

```diff
@@ -36,7 +38,16 @@ public boolean isConnected(Context context) {
             return false;
         }
 
+        int permState = context.checkCallingOrSelfPermission(Manifest.permission.ACCESS_NETWORK_STATE);
+        if (permState != PackageManager.PERMISSION_GRANTED) {
```

Comment:
The permission should be auto-granted to the library when it is listed in the app's manifest. However, in very rare cases (1 in a few million runs), a SecurityException is being thrown (possibly due to a network connectivity flicker right when the app is starting).